index.html

```diff
@@ -0,0 +1,1117 @@
+              <img
+                src="https://d.newsweek.com/en/full/1575559/hand-sanitizer.webp?w=95&h=95&f=af5e9935bbff7640af192a9ea3335c17"
+                alt="" class="w-100 h-100 col-md-4 col-lg-3">
+              <div class="col-lg-9">
+                <h5>Global Coronavirus Death Toll Now Exceeds 11,500</h5>
+                <p>Hospitals struggle to cope in Italy, bars and restaurants in the U.K. have been closed, and the
+                  European
+                  Union allows member states to spend as much as they need to combat the virus.</p>
+              </div>
+            </div>
+            <div class="col-lg-12 d-flex list-item mt-md-5 mb-md-4">
+              <img
+                src="https://d.newsweek.com/en/full/1575683/cameron-van-der-burgh.webp?w=95&h=95&f=f32400a2feb313f893bde85415bf19ee"
+                alt="" class="w-100 h-100 col-md-4 col-lg-3">
+              <div class="col-lg-9">
+                <h5>Olympic Champion Swimmer Diagnosed With Coronavirus Calls It 'Worse Virus'</h5>
+                <p>Cameron van der Burgh, the Olympic gold medalist in the 100-meter breaststroke in 2012 and silver
+                  medalist in 2016, talks about his physical struggles during his 14th day in battling coronavirus.
+                </p>
+              </div>
+            </div>
+            <div class="col-lg-12 d-flex list-item mt-md-5">
+              <img
+                src="https://d.newsweek.com/en/full/1575653/mark-desaulnier.webp?w=95&h=95&f=a5f032524f180168dbb027d89c86a6cd"
+                alt="" class="w-100 h-100 col-md-4 col-lg-3">
+              <div class="col-lg-9">
+                <h5>Congressman Mark DeSaulnier in Critical Condition With Pneumonia</h5>
+                <p>The doctors are doing everything they can to care for the Congressman," the representative's office
+                  said
+                  in a statement.</p>
+              </div>
+            </div>
+          </section>
+        </section>
+      </div>
+      <div class="col-sm-12 d-md-none mt-2 p-0">
+        <section id="top-story-mobile-view" class="col-12 mt-2">
+          <h2>top Story</h2>
+          <div>
+            <img
+              src="https://d.newsweek.com/en/full/1575696/donald-trump.jpg?w=591&h=364&f=c4a7be2fe03204b5fb5327dde00d07c9"
+              alt="" class="w-100">
+            <h3 class="h3-top-story">Trump Announces Activation of the National Guard in Worst-Hit COVID-19 States
+            </h3>
+            <p class="p-top-story">The federal government has deployed hundreds of tons of supplies from our national
+              stockpile to
+              locations
+              with the greatest need in order to assist in those areas," the president said Sunday.</p>
+          </div>
+        </section>
+      </div>
+      <div class="col-lg-4 mt-md-5 mt-lg-1 order-3 ">
+        <section id="opinion" class="col-lg-12 min-content-height sticky-top opinion p-0">
+          <h2>opinion</h2>
```

Comment:
Check the looks of this section. Yours has more spacing between elements.
Yours:
![Screenshot from 2020-04-13 08-01-27](https://user-images.githubusercontent.com/51926937/79122484-5e8dc000-7d5d-11ea-88cf-bf2f731b93f7.png)

Original:
![Screenshot from 2020-04-13 08-01-33](https://user-images.githubusercontent.com/51926937/79122494-62b9dd80-7d5d-11ea-94c9-6ea9a0d3d036.png)